store-failures

```diff
@@ -78,10 +78,20 @@ def main():
         raw_log = url[:-5] if url.endswith(".html") else url
         logging.debug("scanning log %s", raw_log)
 
+        # Parse time from log file name. Wrap in try/except block if the url would be unexpected
+        time = "now"
+        try:
+            parts = url.split("/")[-2].split("-")
+            day = parts[2]
+            hours = parts[3]
+            time = "{0}-{1}-{2} {3}:{4}:{5}".format(day[0:4], day[4:6], day[6:], hours[0:2], hours[2:4], hours[4:])
+        except:
```

Comment:
No blanket exceptions, please; I think what we can reasonably expect is `IndexError`.

Also, I'd rather just store it as a unix timestamp, as a number. Makes age selection/calculation much easier. You can use strftime() for that.

Reply:
changed